### ISSUE TITLE
Can we have control center slider support?

### ISSUE THREAD

**Issue by msdurex** [MonitorControl/MonitorControl#1547]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Is your feature request related to a problem? Please describe

Monitor and Control app works prefect on my external display setup. 

### Describe the solution you'd like

Extra icon on menu bar is too crowded.

### Describe alternatives you've considered

 If it's possible, control center slider support will be so nice.

**Comment by waydabber** [comment]
No, sadly the Control Center is not expandable afaik, it's a locked down part of the OS.
